cmd/integrationArtifactDeploy.go

```diff
@@ -92,14 +94,135 @@ func runIntegrationArtifactDeploy(config *integrationArtifactDeployOptions, tele
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
-		return nil
+		error := PollIFlowDeploymentStatus(retryCount, config, httpClient)
+		return error
 	}
 	responseBody, readErr := ioutil.ReadAll(deployResp.Body)
 
 	if readErr != nil {
-		return errors.Wrapf(readErr, "HTTP response body could not be read, Response status code : %v", deployResp.StatusCode)
+		return errors.Wrapf(readErr, "HTTP response body could not be read, response status code: %v", deployResp.StatusCode)
 	}
-
 	log.Entry().Errorf("a HTTP error occurred! Response body: %v, Response status code : %v", responseBody, deployResp.StatusCode)
-	return errors.Errorf("Integration Flow deployment failed, Response Status code: %v", deployResp.StatusCode)
+	return errors.Errorf("integration flow deployment failed, response Status code: %v", deployResp.StatusCode)
+}
+
+//PollIFlowDeploymentStatus - Poll the integration flow deployment status,retrun status or error details
+func PollIFlowDeploymentStatus(retryCount int, config *integrationArtifactDeployOptions, httpClient piperhttp.Sender) error {
```

Comment:
No need to be exported with capital letter, right?

Reply:
done